Apps/W1/Shopify/app/src/Payments/Tables/ShpfyPaymentTerms.Table.al

```diff
@@ -0,0 +1,78 @@
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Foundation.PaymentTerms;
+
+/// <summary>
+/// Table Shpfy Payment Terms (ID 30157).
+/// </summary>
+table 30157 "Shpfy Payment Terms"
```

Comment:
I've updated the IDs internally as follows:

codeunit 30341 "Shpfy GQL DraftOrderComplete" implements "Shpfy IGraphQL"
codeunit 30355 "Shpfy GQL Fulfill Order" implements "Shpfy IGraphQL"
codeunit 30356 "Shpfy GQL Get Fulfillments" implements "Shpfy IGraphQL"
codeunit 30357 "Shpfy GQL Payment Terms" implements "Shpfy IGraphQL"
codeunit 30358 "Shpfy GQL ShopLocales" implements "Shpfy IGraphQL"
codeunit 30359 "Shpfy GQL TranslationsRegister" implements "Shpfy IGraphQL"
codeunit 30361 "Shpfy Fulfillment API"
codeunit 30362 "Shpfy Posted Invoice Export"
codeunit 30364 "Shpfy Update Sales Invoice"
table 30161 "Shpfy Invoice Header"
codeunit 30360 "Shpfy Payment Terms API"